test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -154,6 +156,8 @@ namespace integration_framework {
                                             block_store_path,
                                             kLocalHost,
                                             torii_port_,
+                                            0,
+                                            "",
```

Comment:
Not cool.
Probably we need some struct like this:
```
struct TlsConfig {
    size_t port,
    std::string key_path
};
```
Then, two initialization parameters can be substituted by the structure.
Moreover, looks like that combined argument can be declared as ``boost::optional`` because it is not a required initialization argument.

Reply:
Fixed in latest version. This struct is available as `iroha::torii::TlsParams`